_docs/unofficial-install-methods/ubuntu-debian-without-root.md

```diff
@@ -11,7 +11,8 @@ First, create a dedicated folder and install The Lounge locally:
 ```sh
 mkdir thelounge
 cd thelounge
-npm install thelounge
+yarn add thelounge
+# npm install thelounge
```

Comment:
Why a commented out version?

Reply:
In the case of no root access, npm is more likely to be installed. I can remove it. Thoughts?